api/admin/controller.py

```diff
@@ -1030,6 +1046,228 @@ def edit_classifications(self, identifier_type, identifier):
 
         return Response("", 200)
 
+    def _validate_cover_image(self, image):
+        image_width, image_height = image.size
+        if image_width < 600 or image_height < 900:
```

Comment:
Please make these into constants.

Reply:
Yes, good idea. I was also wondering if there's anything else I should enforce here like aspect ratio, but since these are uploaded by admins they'll see if the preview looks bad.